psd-web/app/policies/user_policy.rb

```diff
@@ -0,0 +1,5 @@
+class UserPolicy < ApplicationPolicy
+  def export_cases?
+    record.is_psd_admin?
+  end
+end
```

Comment:
I wonder if this would make slightly more sense as part of `InvestigationPolicy`, as it seems to be _about_ the investigations, and pundit policies get passed the current user anyway?

eg something like:

```suggestion
class InvestigationPolicy < Struct.new(:user, :export)
  def export?
      user.is_psd_admin?
    end
end
```

and then in the controller it looks like you can pass the class instead of an instance:

```ruby
 authorize Post, :export?
````

(I had to read [the docs](https://github.com/varvet/pundit#policies) to realise this was an option)

Reply:
Thanks, I was looking for this.